dashboard/app/views/home/_levelbuilder.html.haml

```diff
@@ -0,0 +1,13 @@
+- if current_user && current_user.permissions.where(permission: 'levelbuilder').any?
```

Comment:
Since the UserPermission model is not hooked into the Ability model, I don't think I can use CanCan functionality. Unless I'm missing something?

Note that it may or may not make sense to hook the UserPermission model into CanCan.